spec/services/discogs_requests_service_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe DiscogsRequestsService, type: :service do
+  describe '#search_artist_releases' do
+    it 'gets an artist releases sussessfully' do
+      VCR.use_cassette("get_artist_the_who_releases") do
```

Comment:
Single quotes :nail_care:

Reply:
Also, I think we could use subdirectories here. Something like this:

```ruby
VCR.use_cassette('services/discogs/get_artist')
```

IIRC this should work.